app/bootstrap.php

```diff
@@ -1,14 +1,16 @@
 <?php
 
-require_once dirname(__DIR__) . '/vendor/autoload.php';
+$autoloader = require dirname(__DIR__) . '/vendor/autoload.php';
```

Comment:
Need $autoloader later, and since PHPUnit runs everything in one PHP process (by default) this needs to be changed to a regular require. Which is safe since autoload.php is just a wrapper and the real work is require_once-ed from there.